meta-openpli/recipes-extended/ofgwrite/ofgwrite.bb

```diff
@@ -10,8 +10,9 @@ inherit gitpkgv
 PV = "4.x+git${SRCPV}"
 PKGV = "4.x+git${GITPKGV}"
 
-SRC_URI = "git://github.com/oe-alliance/ofgwrite.git \
-	file://fix_glibc_major.patch"
+SRC_URI = "git://github.com/oe-alliance/ofgwrite.git"
+SRC_URI_append = " file://fix_glibc_major.patch"
+SRC_URI_append_libc-musl = " file://0001-ofgwrite-fix-build-with-musl.patch"
```

Comment:
@andrea-adami Would you please send a PR for https://github.com/oe-alliance/ofgwrite or @betacentauri  could apply this patch.

Reply:
Sure, but here I wanted to ask about   fix_glibc_major.patch: how could it be build for other libc?
I think here we can remove the guards and just  #include <sys/sysmacros.h>

A.A.